controllers/node.go

```diff
@@ -157,7 +157,7 @@ func authenticate(response http.ResponseWriter, request *http.Request) {
 func authorize(hostAllowed, networkCheck bool, authNetwork string, next http.Handler) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		var errorResponse = models.ErrorResponse{
-			Code: http.StatusUnauthorized, Message: logic.Unauthorized_Msg,
+			Code: http.StatusUnauthorized, Message: logic.Forbidden_Msg,
```

Comment:
shouldn't the `Code` match with `Message`?
Ideally, here it should be `Unauthorized_Msg`

Reply:
Fixed.

"is authenticated" doesnt mean the same this as "has access permissions". 401 has a confusing name, [SO thread](https://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses#6937030).